docs/porting/psa/crypto.md

```diff
@@ -0,0 +1,30 @@
+<h2 id="crypto-port">Mbed Crypto</h2>
+
+Mbed OS provides Mbed Crypto for targets that require support for entropy injection. For information about Mbed Crypto, please refer to [the Mbed Crypto repository](https://github.com/ARMmbed/mbed-crypto).
+
+Both targets newly ported to Mbed OS and existing targets that use entropy injection require passing tests to validate Mbed Crypto functionality.
+
+We run tests by default on some supported boards. To run the tests on a custom board, you can also run the tests manually by specifying additional compiler flags from the command-line.

```

Comment:
```suggestion
We run tests by default on a list of supported boards. To run the tests on a custom board, you can also run the tests manually by specifying additional compiler flags from the command-line.
```

Reply:
maybe keep the same as above, 'several'? it will be better to be consistent.